app/src/main/java/de/tum/in/tumcampusapp/activities/CafeteriaActivity.java

```diff
@@ -38,7 +41,10 @@
 
     private ViewPager mViewPager;
     private int mCafeteriaId = -1;
-    private List<Cafeteria> mCafeterias;
+    private CafeteriaViewModel cafeteriaViewModel;
+    private List<Cafeteria> mCafeterias = new LinkedList<>();
```

Comment:
There's really never a case, where a LinkedList makes sense. Please default to an ArrayList